Formula/schismtracker.rb

```diff
@@ -11,16 +12,11 @@ class Schismtracker < Formula
     sha256 "f892d60e3e74f352d7d9efbbf84b6470e4e751ffafe55c9b6ab9913bd611dba2" => :mavericks
   end
 
-  head do
-    url "http://schismtracker.org/hg/", :using => :hg
-
-    depends_on "autoconf" => :build
-  end
-
+  depends_on "autoconf" => :build
```

Comment:
Like I mentioned in the other PR, `autoconf` is actually only used by the HEAD build, so it should stay in the `head do` block, just like the HEAD `url` should.